crates/iota-protocol-config/src/lib.rs

```diff
@@ -1879,14 +1890,16 @@ impl ProtocolConfig {
                     cfg.validator_validate_metadata_cost_base = Some(20000);
                 }
                 6 => {
-                    // TODO: add new consensus related config params to this
-                    // version
-                    // Enable round prober in consensus.
-                    cfg.feature_flags.consensus_round_prober = true;
-                    cfg.feature_flags
-                        .consensus_distributed_vote_scoring_strategy = true;
-
                     cfg.feature_flags.variant_nodes = true;
+                    if chain != Chain::Mainnet {
+                        // Enable zstd compression for consensus in testnet
+                        cfg.feature_flags.consensus_zstd_compression = true;
+                        // Enable round prober in consensus.
+                        cfg.feature_flags.consensus_round_prober = true;
+                        // Enable distributed vote scoring.
+                        cfg.feature_flags
+                            .consensus_distributed_vote_scoring_strategy = true;
```

Comment:
```suggestion
                        // Enable round prober in consensus.
                        cfg.feature_flags.consensus_round_prober = true;
                        // Enable distributed vote scoring.
                        cfg.feature_flags
                            .consensus_distributed_vote_scoring_strategy = true;
                        // Enable zstd compression for consensus in testnet
                        cfg.feature_flags.consensus_zstd_compression = true;
```

same order as they were declared.